custom/AggregatedFormatter.py

```diff
@@ -0,0 +1,16 @@
+
+from AbstractCustomFormatter import AbstractCustomFormatter
+from io import StringIO
+
+class AggregatedFormatter(AbstractCustomFormatter):
+
+    def __init__(self, formatters):
+        self.formatters = formatters
+
+    def format_lines(self, lines):
+        if len(self.formatters) == 0:
+            return "".join(lines)
```

Comment:
nit: should this just be an error instead of ignoring it? seems like there should be at least one formatter if using this type